hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -141,27 +141,18 @@ public void updateCongestionStartsFrom(FeeMultiplierSource feeMultiplierSource)
 		if (congestionStarts.length == 0) {
 			congestionLevelStarts = NO_CONGESTION_STARTS;
 		} else {
-			congestionLevelStarts = new RichInstant[congestionStarts.length];
-			for (int i = 0; i < congestionStarts.length; i++) {
-				congestionLevelStarts[i] = RichInstant.fromJava(congestionStarts[i]);
-			}
+			congestionLevelStarts = congestionStarts;
```

Comment:
Line 140 is not shown here. But 
https://github.com/hashgraph/hedera-services/blob/d1409172a44097bc4cfd7e67e14b2f835621d46a/hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java#L140
can `feeMultiplierSource.congestionLevelStarts()` return a `null` instead of an array of `Instant`?

Anyway, it's always better to check if the return value is a `null` or not. So the correct change here should be
```
if (null == congestionStarts) {
	congestionLevelStarts = NO_CONGESTION_STARTS;
} else {
	congestionLevelStarts = congestionStarts;
}
```

Reply:
It's expected FeeMultiplerSource.congestionLevelStarts() should always return an Array of `Instant`, even if it can be of size 0. So far this contract is respected.
That said, it won't hurt to always check the `null` return from `feeMultiplierSource.congestionLevelStarts()`.